sdk/python/feast/feature_server.py

```diff
@@ -58,6 +67,22 @@ def get_online_features(body=Depends(get_body)):
             # Raise HTTPException to return the error message to the client
             raise HTTPException(status_code=500, detail=str(e))
 
+    @app.post("/write-to-online-store")
+    def write_to_online_store(body=Depends(get_body)):
+        try:
+            request = WriteToFeatureStoreRequest(**json.loads(body))
```

Comment:
Think this is generally fine, though we'd probably also want to have some warning that this is experimental since we may want to expose a slightly different endpoint to allow for streaming transformations, or pushing directly to the offline store. 


An example warning might be e.g.

```python
warnings.warn(
    "write_to_online_store is an experimental feature. "
    "This API is unstable and it could be changed in the future. "
    "We do not guarantee that future changes will maintain backward compatibility.",
    RuntimeWarning,
)
```

Reply:
Done ✅